modules/021-cni-cilium/hooks/enable_node_routes.go

```diff
@@ -39,6 +39,11 @@ func enableNodeRoutes(input *go_hook.HookInput) error {
 // use value from config
 // if not present, enable everywhere except clouds (other than openstack and vsphere)
 func shouldEnableNodeRoutes(input *go_hook.HookInput) bool {
+	// Works only with `tunnelMode: Disabled`
+	if input.ConfigValues.Get("cniCilium.tunnelMode").String() == "VXLAN" {
```

Comment:
I think that this is a bad ux example. Instead of two settings: createNodeRoutes and tunnelMode. It's better to have just one, like mode:
```
  mode:
    type: string
    enum: ["Direct", "DirectWithNodeRoutes", "VXLAN"]
    default: "Direct"
```